### DISCUSSION TITLE
Package registration

### DISCUSSION THREAD

**Discussion by ghost** [inertiajs/inertia#822] [Help] [answered]
Hi developers! Asking for help with register vue package in Inertia JS. The basic init code of Inertia is:

<img width="378" alt="Снимок экрана 2021-07-24 в 15 57 10" src="https://user-images.githubusercontent.com/77324079/126869104-37f92033-a809-487b-a16f-14b3a2f9b4c6.png">

I just need to register Vue-Toastification plugin, but have no idea what to do. I try import package before and call it inside use(), but nothing happened. What the secret is?

**Comment by ajnsn** [comment, marked as answer]
@Macartini Just add it to the `createApp` chain as `.use(Toast, options)`, e.g. 

```js
import Toast from 'vue-toastification'
const options = {
    // You can set your default options here
};

...
  setup({ el, App, props, plugin }) {
    createApp({ render: () => h(App, props) })
      .use(plugin)
      .use(Toast, options)
      .mount(el)
  },
...
```

Taken from [Vue Toastification (for Vue 3) Plugin registration](https://github.com/Maronato/vue-toastification/tree/next#plugin-registration)

**Reply by reinink** [reply]
Thanks for your continued help here @ajnsn!
